tests/models/test_mappedoperator.py

```diff
@@ -57,6 +59,32 @@ def test_task_mapping_with_dag():
     assert mapped.downstream_list == [finish]
 
 
+def test_task_mapping_with_dag_and_list_of_pandas_dataframe(caplog):
+    caplog.set_level(logging.INFO)
+
+    class UnrenderableClass:
+        def __non_zero__(self):
+            raise ValueError("Similar to Pandas DataFrames, this class does not implement zero comparison.")
+
+    class CustomOperator(BaseOperator):
+        template_fields = ("arg",)
+
+        def __init__(self, arg, **kwargs):
+            super().__init__(**kwargs)
+            self.arg = arg
+
+        def execute(self, context: Context):
+            pass
+
+    with DAG("test-dag", start_date=DEFAULT_DATE) as dag:
+        task1 = CustomOperator(task_id="op1", arg=None)
+        unrenderable_values = [UnrenderableClass(), UnrenderableClass()]
+        mapped = CustomOperator.partial(task_id="task_2").expand(arg=unrenderable_values)
+        task1 >> mapped
+    dag.test()
+    assert caplog.text.count("task_2 ran successfully") == 2
```

Comment:
The test seems to pass without the change.

Reply:
Just fixed, @ephraimbuddy, it stopped working after I removed the Pandas dependency - I had misspelt `__nonzero__` as `__non_zero__`.

Reply:
Looks like it's still there. I think `__bool__` will raise